<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GapDetectionDialogBase</class>
 <widget class="QDialog" name="GapDetectionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hop Gap Detection</string>
  </property>
  <widget class="QPushButton" name="pbHello">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLayers">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbLayers_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Raster</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shape file of area</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLoadFile">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LoadFile</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
